src/components/package/edit-managed/managed-package-edit.js

```diff
@@ -54,13 +53,11 @@ class ManagedPackageEdit extends Component {
     let { router } = this.context;
 
     if ((nextProps.initialValues.isSelected !== this.props.initialValues.isSelected) ||
-    (nextProps.initialValues.isVisible !== this.props.initialValues.isVisible) ||
-    (nextProps.initialValues.packageAllowedToAddTitles !== this.props.initialValues.packageAllowedToAddTitles)) {
+    (nextProps.initialValues.isVisible !== this.props.initialValues.isVisible)) {
       this.setState({
         ...this.state,
         packageSelected: nextProps.initialValues.isSelected,
-        packageVisible: nextProps.initialValues.isVisible,
-        packageAllowedToAddTitles: nextProps.initialValues.allowKbToAddTitles
+        packageVisible: nextProps.initialValues.isVisible
```

Comment:
This looks good to me - so I assume once the other toggles are moved over to radios - we may be able to get rid of these other state settings as well?

Reply:
Yess! I'm excited about that :D